src/main/java/io/github/noeppi_noeppi/libx/data/provider/ItemModelProviderBase.java

```diff
@@ -26,7 +26,7 @@
 
     public static final ResourceLocation GENERATED = new ResourceLocation("item/generated");
     public static final ResourceLocation HANDHELD = new ResourceLocation("item/handheld");
-    public static final ResourceLocation TEISR_PARENT = new ResourceLocation(LibX.getInstance().modid, "item/base/teisr");
+    public static final ResourceLocation BEISR_PARENT = new ResourceLocation(LibX.getInstance().modid, "item/base/beisr");
```

Comment:
Should either be named `bewlr` for BlockWithoutLevelRenderer or `special_block` which might be more meaningful.